mods/ra/maps/red-zone/red-zone.lua

```diff
@@ -0,0 +1,608 @@
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+PlayerInfo = { }
+TankCamera =
+{
+	easy = "CAMERA.15_Cells",
+	normal = "CAMERA.12_Cells",
+	hard = "CAMERA.9_Cells"
+}
+
+AirBaseCamera =
+{
+	easy = "CAMERA.20_Cells",
+	normal = "CAMERA.17_Cells",
+	hard = "CAMERA.14_Cells"
+}
+
+PrisonBaseCamera =
+{
+	easy = "CAMERA.24_Cells",
+	normal = "CAMERA.21_Cells",
+	hard = "CAMERA.18_Cells"
+}
+
+CommunicationsCenterCamera =
+{
+	easy = "CAMERA.27_Cells",
+	normal = "CAMERA.24_Cells",
+	hard = "CAMERA.21_Cells"
+}
+
+MainBaseCamera =
+{
+	easy = "CAMERA.13_Cells",
+	normal = "CAMERA.11_Cells",
+	hard = "CAMERA.9_Cells"
+}
+
+MainBaseCameraMedium =
+{
+	easy = "CAMERA.17_Cells",
+	normal = "CAMERA.15_Cells",
+	hard = "CAMERA.13_Cells"
+}
+
+MainBaseCameraLarge =
+{
+	easy = "CAMERA.24_Cells",
+	normal = "CAMERA.22_Cells",
+	hard = "CAMERA.20_Cells"
+}
+
+MainBaseCameraEntry =
+{
+	easy = "CAMERA.19_Cells",
+	normal = "CAMERA.17_Cells",
+	hard = "CAMERA.15_Cells"
+}
+
+MainBaseCamera2 =
+{
+	easy = "CAMERA.18_Cells",
+	normal = "CAMERA.16_Cells",
+	hard = "CAMERA.14_Cells"
+}
+
+Difficulty = Map.LobbyOption("difficulty")
+
+APCInsertionPath = { APCEntryPoint.Location, APCDropoffPoint.Location }
+
+BackupHeliPath = { SpawnPoint1.Location, LZ2.Location }
+BackupHeliPathStrike = { SpawnPoint1.Location, LZ.Location }
+SovietReinforcements =
+{
+	easy = { "e1", "e1", "e1", "e1", "e3", "e3", "e3", "e3" },
+	normal = { "e1", "e1", "e1", "e1", "e3", "e3", "e3" },
+	hard = { "e1", "e1", "e1", "e1", "e3", "e3" }
+}
+
+HindUnit1 = { Hind1 }
+HindUnit2 = { Hind2 }
+HindUnit3 = { Hind3 }
+
+PatrolInfantry = { EnglandAirBasePatrolInfantry1, EnglandAirBasePatrolInfantry2, EnglandAirBasePatrolInfantry3 }
+InfantryPath = { PatrolPoint12.Location, PatrolPoint13.Location, PatrolPoint14.Location }
+
+PatrolDestroyers1 = { Destroyer2, Destroyer1 }
+PatrolDestroyers2 = { Destroyer4, Destroyer5 }
+DestroyerPath = { PatrolPoint1.Location, PatrolPoint7.Location, PatrolPoint8.Location }
+
+NukeTimer =
+{
+	easy = DateTime.Minutes(30),
+	normal = DateTime.Minutes(25),
+	hard = DateTime.Minutes(20)
+}
+
+DemoTruckInsertionPath = { TransportInsertionPoint.Location, TransportDropoffPoint.Location }
+
+APCDropoff = function()
+	Media.PlaySpeechNotification(ussr, "ReinforcementsArrived")
+
+	local passengers = Reinforcements.ReinforceWithTransport(ussr, "apc.entryunit", { "thf" }, APCInsertionPath, { APCExitPoint.Location })[2]
+	local thief = passengers[1]
+
+	Trigger.OnKilled(thief, MissionFailed)
+	Trigger.AfterDelay(DateTime.Seconds(6), function()
+		desertPatrolCam = Actor.Create(TankCamera[Difficulty], true, { Owner = ussr, Location = DesertCam.Location })
+	end)
+
+	if ussr.HasNoRequiredUnits() then
+		MissionFailed()
+	end
+end
+
+EnglandAirBaseReveal = function()
+	if not EnglandAirBaseRevealed then
+		Media.DisplayMessage("Hijack a Hind Helicopter.")
```

Comment:
Considering we changed these to Blackhawks for Skirmish/Multiplayer I'd go ahead a switch this to the UH60.